Easydict/Feature/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -527,6 +527,7 @@ - (void)focusInputTextView {
         [NSApp activateIgnoringOtherApps:YES];
         
         [self.baseQueryWindow makeFirstResponder:self.queryView.textView];
+        self.queryView.textView.selectedRange = NSMakeRange(0, self.queryText.length);
```

Comment:
```suggestion
        self.queryView.textView.selectedRange = NSMakeRange(0, self.inputText.length);
```

Note that sometimes `inputText` is different from `queryText` if there is a comment `//`.

<img width="455" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/675ceeb7-8b11-4bcd-aa24-d7abf7a8ba7d">

Reply:
got it